lib/util/cache/repository/impl/local.spec.ts

```diff
@@ -84,8 +87,8 @@ describe('util/cache/repository/impl/local', () => {
     );
 
     await localRepoCache.load();
-
```

Comment:
I'm trying to structure my tests according to Arrange-Act-Assert triad, that's why I'm adding these spaces.

Reply:
That's my IDE does its auto formatting thing 😅.

p.s. That's sounds like a good practice, ill adopt it as well thanks for the clarification